tests/lib/rules/require-meta-fixable.js

```diff
@@ -207,5 +232,27 @@ ruleTester.run('require-meta-fixable', rule, {
       `,
       errors: [{ messageId: 'missing', type: 'Identifier' }],
     },
+
+    // catchNoFixerButFixableProperty = true
+    {
+      code: `
+        module.exports = {
+          meta: { fixable: 'code' },
+          create(context) { context.report({node, message}); }
+        };
+      `,
+      options: [{ catchNoFixerButFixableProperty: true }],
+      errors: [{ messageId: 'noFixerButFixableValue', type: 'Literal' }],
+    },
+    {
+      code: `
+        module.exports = {
+          meta: { fixable: 'whitespace' },
+          create(context) { context.report({node, message}); }
+        };
+      `,
+      options: [{ catchNoFixerButFixableProperty: true }],
+      errors: [{ messageId: 'noFixerButFixableValue', type: 'Literal' }],
+    },
```

Comment:
can you add some tests that sometimes do not have a fixer - it seems a quite common case?
e.g.
```js
// valid
module.exports = {
  meta: { fixable: "whitespace" },
  create(context) {
    foo
      ? context.report({ node, message, fix })
      : ontext.report({ node, message });
  },
};

// invalid
module.exports = {
  meta: { fixable: null },
  create(context) {
    foo
      ? context.report({ node, message, fix })
      : ontext.report({ node, message });
  },
};

```

Reply:
Sure, added tests.